RELEASE_NOTES.md

```diff
@@ -2,13 +2,14 @@
 
 ## Individual Updates
 
-- [#361](https://github.com/IAMconsortium/pyam/pull/361) iam-units refactored from a Git submodule to a Python dependency of pyam.
+- [#361](https://github.com/IAMconsortium/pyam/pull/361),
+  [#369](https://github.com/IAMconsortium/pyam/pull/369) iam-units refactored from a Git submodule to a Python dependency of pyam.
```

Comment:
please order in descending order and add description like "bugfix to allow same-species mass conversion"

Reply:
Done!